src/game/game_test.go

```diff
@@ -0,0 +1,24 @@
+package game
+
+import (
+    "testing"
+)
+
+func TestDeserialize(t *testing.T) {
```

Comment:
There is no assertion in your test. It's better in my opinion to directly know what is tested.

For example:

`it should throw an error when the board doesn't contain 6 columns`
`it should deserialize the board into an array of arrays`